aten/src/ATen/native/native_functions.yaml

```diff
@@ -6642,6 +6642,16 @@
   structured_delegate: _addmm_activation.out
   variants: function, method
 
+- func: _scaled_mm(Tensor self, Tensor mat2, *, ScalarType? dtype=None, Tensor? scale_a=None, Tensor? scale_b=None, Tensor? scale_result=None) -> (Tensor, Tensor)
```

Comment:
nit: you can name these outputs so that they will be named on the python side when returned automatically!

Reply:
I though this is a discouraged behavior (as it requires allowlisting it)

Reply:
Yeah that is surprising...  not needed here for sure!